buf.yaml

```diff
@@ -1,7 +1,10 @@
-version: v1beta1
-build:
-  roots:
-    - .
+version: v1
+name: buf.build/odpf/proton
+deps:
+  - buf.build/googleapis/googleapis
+  - buf.build/grpc-ecosystem/grpc-gateway
 breaking:
   use:
-    - WIRE
\ No newline at end of file
+    - WIRE
+  except:
+    - RESERVED_MESSAGE_NO_DELETE
```

Comment:
openapiv2 from grpc-gateway had some changes that are not compatible with this rule. We can remove this rule going forward.

Basically they had reserved fields so they could use them later, and now they implemented those.